host_cleanup.sh

```diff
@@ -31,40 +31,37 @@ if [ "${CAPM3_RUN_LOCAL}" = true ]; then
   fi
 fi
 
-
 ANSIBLE_FORCE_COLOR=true ansible-playbook \
-    -e "working_dir=$WORKING_DIR" \
-    -e "num_nodes=$NUM_NODES" \
-    -e "extradisks=$VM_EXTRADISKS" \
-    -e "virthost=$HOSTNAME" \
-    -e "manage_baremetal=$MANAGE_BR_BRIDGE" \
-    -e "nodes_file=$NODES_FILE" \
-    -i vm-setup/inventory.ini \
-    -b -v vm-setup/teardown-playbook.yml
+  -e "working_dir=$WORKING_DIR" \
+  -e "num_nodes=$NUM_NODES" \
+  -e "extradisks=$VM_EXTRADISKS" \
+  -e "virthost=$HOSTNAME" \
+  -e "manage_baremetal=$MANAGE_BR_BRIDGE" \
+  -e "nodes_file=$NODES_FILE" \
+  -i vm-setup/inventory.ini \
+  -b -v vm-setup/teardown-playbook.yml
 
-if [ "$USE_FIREWALLD" == "False" ]; then
- ANSIBLE_FORCE_COLOR=true ansible-playbook \
-    -e "{use_firewalld: $USE_FIREWALLD}" \
-    -e "external_subnet_v4: ${EXTERNAL_SUBNET_V4}" \
-    -e "firewall_rule_state=absent" \
-    -i vm-setup/inventory.ini \
-    -b -v vm-setup/firewall.yml
-fi
+ANSIBLE_FORCE_COLOR=true ansible-playbook \
+  -e "{use_firewalld: $USE_FIREWALLD}" \
+  -e "external_subnet_v4: ${EXTERNAL_SUBNET_V4}" \
```

Comment:
This is not JSON nor Bash format, and doesn't work. `external_subnet_v4` extra was also removed in Adam's patch, could it be removed here too?

Reply:
I think these changes should be addressed only in one PR and then rebase the second PR once the first merged I do not want to replicate adams changes here else his PR will be meaningless ?

Reply:
@Rozzii 's PR is now merged, please rebase.

Reply:
rebased